### PR TITLE
Add 6.0 Release Notes for Action Pack

### PR DESCRIPTION
**Pull request by alexcameron89**
### Summary

I've gone through the CHANGELOG for Action Pack, pulled out anything
that was backported, rephrased some of the descriptions and found their
PR's/commits as links.

If there is anything that you feel I should remove, please let me know. 😄

### CODE REVIEW

File: guides/source/6_0_release_notes.md [hunk 1]
```diff
@@ -178,10 +178,82 @@ Please refer to the [Changelog][action-pack] for detailed changes.
...
+    ([Pull Request](https://github.com/rails/rails/pull/35081))
+
+*   Add `ActionDispatch::HostAuthorization` middleware that guards against DNS rebinding
+    attacks.
+    ([Pull Request](https://github.com/rails/rails/pull/33145))
+
+*   Allow the use of `parsed_body` in `ActionController::TestCase`.
+    ([Pull Request](https://github.com/rails/rails/pull/34717))
+
+*   Raise an error on root route naming conflicts.
```
Comment (prathamesh-sonpatki) [inline guides/source/6_0_release_notes.md]: We can expand on this, perhaps something like ~ 

`Raise an error when a root route exists and other is defined without name.`

File: guides/source/6_0_release_notes.md [hunk 2]
```diff
@@ -178,10 +178,82 @@ Please refer to the [Changelog][action-pack] for detailed changes.
...
 
+*   Deprecate `ActionDispatch::Http::ParameterFilter` in favor of `ActiveSupport::ParameterFilter`.
+    ([Pull Request](https://github.com/rails/rails/pull/34039))
+
+*   Deprecate controller level `force_ssl` in favor of `config.force_ssl`.
+    ([Pull Request](https://github.com/rails/rails/pull/32277))
+
 ### Notable changes
 
+*   Introduce Actionable Errors, allowing the ability to dispatch actions from
```
Comment (rafaelfranca) [inline guides/source/6_0_release_notes.md]: This is internal of the framework so I think we should not include in the changelog.

File: guides/source/6_0_release_notes.md [hunk 3]
```diff
@@ -178,10 +178,82 @@ Please refer to the [Changelog][action-pack] for detailed changes.
...
+    attacks.
+    ([Pull Request](https://github.com/rails/rails/pull/33145))
+
+*   Allow the use of `parsed_body` in `ActionController::TestCase`.
+    ([Pull Request](https://github.com/rails/rails/pull/34717))
+
+*   Raise an error on root route naming conflicts.
+    ([Pull Request](https://github.com/rails/rails/pull/34494))
+
+*   Allow the use of `#rescue_from` for parameter parse errors.
```
Comment (prathamesh-sonpatki) [inline guides/source/6_0_release_notes.md]: Can we say `for handling parameter parsing errors`.

**Review by prathamesh-sonpatki — CHANGES_REQUESTED** [review]

File: guides/source/6_0_release_notes.md (hunk 3 above)
Comment (alexcameron89) [reply guides/source/6_0_release_notes.md]: 👍 Got it!

File: guides/source/6_0_release_notes.md (hunk 1 above)
Comment (alexcameron89) [reply guides/source/6_0_release_notes.md]: How do you like `Raise an `ArgumentError` when multiple root routes exists in the same context without `as:` naming specifications.`?

**Comment by alexcameron89** [comment]
Thank you @rafaelfranca and @prathamesh-sonpatki, I've made those changes. 😄

**Review by rafaelfranca — APPROVED** [review]

**Review by prathamesh-sonpatki — APPROVED** [review]

**Pull request merged**